ocaml/xapi/cli_operations.ml

```diff
@@ -167,7 +169,9 @@ let diagnostic_net_stats printer rpc session_id params =
   let uri (_, u, _) =
     not (List.mem_assoc "uri" params)
     || (String.lowercase_ascii u = String.lowercase_ascii (List.assoc "uri" params)) in
-  let has_param x = not(List.mem_assoc "params" params) || (List.mem x (String.split ',' (List.assoc "params" params))) in
+  let has_param x =
+    not (List.mem_assoc "params" params)
+    || (List.mem x (String.split_on_char ',' (List.assoc "params" params))) in
```

Comment:
can this use List.assoc_opt here instead of mem+assoc?

Reply:
Can't, this `List` is from stdext, I'll add it to https://github.com/xapi-project/xen-api/pull/4038 I already have removed Listext for this reason.

Reply:
See https://github.com/xapi-project/xen-api/commit/acd69b306